From 5c6fb914e68625fbec8f413bbee7dcd707e80c59 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Wed, 30 Nov 2011 23:52:28 +0900
Subject: [PATCH 4/5] copyright-format: correct or add links to SPDX.

Wording: Charles Plessy <plessy@debian.org>
Seconded: gregor herrmann <gregoa@debian.org>
Closes: #641071
---
 copyright-format/copyright-format.xml |   29 ++++++++++++++---------------
 1 files changed, 14 insertions(+), 15 deletions(-)

diff --git a/copyright-format/copyright-format.xml b/copyright-format/copyright-format.xml
index 4c18c3b..e854c2a 100644
--- a/copyright-format/copyright-format.xml
+++ b/copyright-format/copyright-format.xml
@@ -580,8 +580,8 @@ License: MPL-1.1
               </entry>
               <entry>
                 Apache license
-                <ulink url="http://spdx.org/licenses/ASL-1.0">1.0</ulink>,
-                <ulink url="http://spdx.org/licenses/ASL-2.0">2.0</ulink>.
+                <ulink url="http://spdx.org/licenses/Apache-1.0">1.0</ulink>,
+                <ulink url="http://spdx.org/licenses/Apache-2.0">2.0</ulink>.
               </entry>
             </row>
             <row>
@@ -710,9 +710,8 @@ License: MPL-1.1
                 CC0
               </entry>
               <entry>
-                Creative Commons <ulink
-                url="http://creativecommons.org/license/zero/">Universal
-                waiver</ulink>.
+                Creative Commons Zero 1.0 Universal
+                <ulink url="http://spdx.org/licenses/CC0-1.0">1.0</ulink>.
               </entry>
             </row>
             <row>
@@ -720,8 +719,8 @@ License: MPL-1.1
                 CDDL
               </entry>
               <entry>
-                <ulink url="http://spdx.org/licenses/CDDL">Common Development
-                and Distribution License</ulink>.
+                Common Development and Distribution License
+                <ulink url="http://spdx.org/licenses/CDDL-1.0">1.0</ulink>.
               </entry>
             </row>
             <row>
@@ -772,7 +771,6 @@ License: MPL-1.1
                 <ulink url="http://spdx.org/licenses/LGPL-2.1">2.1</ulink>,
                 <ulink url="http://spdx.org/licenses/LGPL-3.0">3.0</ulink>, or
                 GNU Library General Public License
-                <ulink url="http://spdx.org/licenses/LGPL-1.0">1.0</ulink>,
                 <ulink url="http://spdx.org/licenses/LGPL-2.0">2.0</ulink>.
               </entry>
             </row>
@@ -781,8 +779,8 @@ License: MPL-1.1
                 GFDL
               </entry>
               <entry>
-                <ulink url="http://spdx.org/licenses/FDL-1.0">GNU Free
-                Documentation License</ulink>.
+                GNU Free Documentation License 1.0, or
+                <ulink url="http://spdx.org/licenses/GFDL-1.1">1.1</ulink>.
               </entry>
             </row>
             <row>
@@ -827,12 +825,13 @@ License: MPL-1.1
             </row>
             <row>
               <entry>
-                Python-CNRI
+                Python
               </entry>
               <entry>
-                <ulink url="http://spdx.org/licenses/Python-CNRI">Python
-                license</ulink>.
+                Python license
+                <ulink url="http://spdx.org/licenses/Python-2.0">2.0</ulink>.
               </entry>
+              <!-- See https://fossbazaar.org/pipermail/spdx-legal/2011-February/000010.html -->
             </row>
             <row>
               <entry>
@@ -869,8 +868,8 @@ License: MPL-1.1
                 Zope
               </entry>
               <entry>
-                Zope Public License
-                  <ulink url="http://spdx.org/licenses/ZPL-1.0">1.0</ulink>,
+                Zope Public License 1.0,
+                  <ulink url="http://spdx.org/licenses/ZPL-1.1">1.1</ulink>,
                   <ulink url="http://spdx.org/licenses/ZPL-2.0">2.0</ulink>,
                   <ulink url="http://spdx.org/licenses/ZPL-2.1">2.1</ulink>.
               </entry>
-- 
1.7.7.3

